packages/components/organisms/f-header/src/components/Logo.vue

```diff
@@ -41,6 +45,10 @@ export default {
             type: String,
             required: true
         },
+        linkDisabled: {
+            type: Boolean,
+            default: false
```

Comment:
I'd make this `required: true` rather than `default: false`, as the prop at the external interface level is already optional, so we'll be supplying the value in from there.

Reply:
Makes sense. Done.

Reply:
If we do this; we would need to make this a major release; as it would contain breaking functionality.